packages/components/RoadmapSection.css

```diff
@@ -0,0 +1,31 @@
+.roadmap-card-blue {
+  border-color: #17a9bc;
+}
+
+.roadmap-card-blue:hover {
+  box-shadow: -8px 8px 0 #17a9bc;
+}
+
+.roadmap-card-orange {
+  border-color: #f99223;
+}
+
+.roadmap-card-orange:hover {
+  box-shadow: -8px 8px 0 #f99223;
+}
+
+.roadmap-card-pink {
+  border-color: #ff4cff;
+}
+
+.roadmap-card-pink:hover {
+  box-shadow: -8px 8px 0 #ff4cff;
+}
+
+.roadmap-card-purple {
+  border-color: #b794f4;
+}
+
+.roadmap-card-purple:hover {
+  box-shadow: -8px 8px 0 #b794f4;
+}
```

Comment:
Lets use css variables:
- https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

With this, we can reuse the same styles, but changing the color locally

```css
.card-blue {
  --color: #17a9bc;
}

.card-orange {
  --color: #f99223;
}

.card-pink {
  --color: #ff4cff;
}

.card-purple {
  --color: #b794f4;
}

.roadmap-card {
  border-color: var(--color);
}

.roadmap-card:hover {
  box-shadow: -8px 8px 0 var(--color);
}
```

Then I'd change the card classes to something like:

```js
      <div
        className={classes(
          "flex justify-center rounded-lg border-2 py-8 transition-shadow duration-300 ease-in-out",
          `roadmap-card ${colorClass}`, // this should be ok, as `colorClass` cannot be null
        )}


```

Reply:
Made in: https://github.com/podcodar/webapp/pull/211/commits/bf93f460f66af5320da736f1b5bd4e5e134cb16e and https://github.com/podcodar/webapp/pull/211/commits/8ea13d4ef97aea2b97f97ee77ef0217d08c292b3